content/blog/2025-09-10-numbered-databases/index.md

```diff
@@ -0,0 +1,92 @@
++++
+title= "Numbered “Databases” in Valkey 9.0"
+description = "If you explore Valkey’s documentation you might run across a tantalizing feature called ‘numbered databases’ which allows you to separate the keyspace into (by default) 16 different databases. "
```

Comment:
I would suggest include the point of this blog in the description. Maybe something in the lines of:
"Unlike it's predecessor, Valkey continue to invest in extending this 'namespacing' ability.   In this blog we will describe the  'numbered databases' in Valkey. We will discuss different use cases and what problems are not solved using numbered databases. Finally we will go over the latest changes introduced in Valkey 9.0, which allows valkey to scale with multiple databases used."